bot.py

```diff
@@ -140,63 +160,90 @@ def pr_url(self):
         Merge the release candidate into the release branch, tag it, merge to master, and wait for deployment
         """
         check_call([in_script_dir("finish_release.sh"), self.repo_dir, self.version])
-        self.say("Merged evil scheme {}! Now deploying to production...".format(self.version))
+        await self.say("Merged evil scheme {}! Now deploying to production...".format(self.version))
         check_call([in_script_dir("wait_for_deploy.sh"), self.repo_dir, self.prod_hash_url, "release"])
-        self.say(
+        await self.say(
             "My evil scheme {} has been released to production. "
             "And by 'released', I mean completely...um...leased.".format(self.version)
         )
 
-    def message_if_unchecked(self):
+    async def message_if_unchecked(self):
         """
         Send a message next morning if any boxes are not yet checked off
         """
         unchecked_authors = get_unchecked_authors(self.org, self.repo, self.version)
         if unchecked_authors:
             slack_usernames = self.translate_slack_usernames(unchecked_authors)
-            self.say(
+            await self.say(
                 "What an unexpected surprise! "
                 "The following authors have not yet checked off their boxes: {}".format(
                     ", ".join(slack_usernames)
                 )
             )
 
+    async def delay_message(self):
+        now = datetime.now()
+        tomorrow_at_10 = next_workday_at_10(now)
+        await asyncio.sleep((tomorrow_at_10 - now).total_seconds())
+        await self.message_if_unchecked()
+
+
+def get_version_number(message):
+    return re.findall('[0-9.]+', message)
+
+mm_urls = {
+    'repo_dir': os.environ.get('REPO_DIR'),
```

Comment:
This will get executed at the time of `import` rather than when `main` is run. Can we move all of this into `main` and pass it around as an argument as necessary?